openid-sharedsignals-framework-1_0.md

```diff
@@ -165,15 +165,15 @@ and Coordination (RISC) and the Continuous Access Evaluation Profile ({{CAEP}})
 
 This specification defines:
 
-* A profile for Security Events Tokens {{RFC8417}}
+* A Profile for Security Events Tokens {{RFC8417}}
```

Comment:
I'm not sure "Profile" is appropriate here, because it's "a profile", which is a generic term.